modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -230,7 +230,8 @@ Configuration properties can be overwritten. The Hierarchy in which these are tr
 . Environment Properties
 . Deployment Properties
 . Application Properties
+. Global Properties
 
-So, for example, if a configuration property is defined in a System Property, and there is also an application configuration property, the value for that application is the last one.
+So, for example, if a configuration property is defined in a System Property, and there is also an application configuration property, the value for that application is the first one.
```

Comment:
Not sure this is clear for the end user. Is the concept of **application configuration property** clear?. Also instead of talking about the **first one** I would talk about precedence.

Reply:
```suggestion
So, for example, if a configuration property is defined in a System Property, and there is also an application configuration property, the value for that application is the most precedence (in this case, the System Property).
```